packages/common-utils/src/queryParser.ts

```diff
@@ -476,7 +549,8 @@ export class CustomSchemaSQLSerializerV2 extends SQLSerializer {
 
       return {
         column: this.implicitColumnExpression,
-        propertyType: 'string' as const,
+        columnJSON: undefined,
+        propertyType: JSDataType.String as const,
```

Comment:
nit: no need for `as const`

Reply:
When I remove it get ts syntax error, looks like it treat it as string, so still need `as const` at here.

Reply:
Hmm...it ts gets confused. the type for `propertyType` might be too strict here (https://github.com/hyperdxio/hyperdx/blob/52572b40bb1dbe83e77b7b0f7056cb977d1c5fb7/packages/common-utils/src/queryParser.ts#L185-L190). Changing it to `propertyType?: JSDataType;` should solve the problem